mantidproject/mantid
# Install missing python files

Two recently added python files never made it into install packages.

**To test:**

Download an rpm (or make one yourself) then run

``` shell
$ rpm -qilp mantid*-3.7*.rpm  | grep pymantidplot
```

In the result you should see a `pymantidplot/mpl/__init__.py` and `pymantidplot/mpl/backend_mtdqt4agg.py`.

_There is no associated issue_

_Does not need to be in the release notes_ because the bug was never in a release

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
